<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlItem</class>
 <widget class="QWidget" name="ControlItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="hLabel">
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sLabel">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="sDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="vLabel">
       <property name="text">
        <string>v</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="vDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="hBaseLabel">
       <property name="text">
        <string>hBase</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="hBaseDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="sBaseLabel">
       <property name="text">
        <string>sBase</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="sBaseDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="vBaseLabel">
       <property name="text">
        <string>vBase</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="vBaseDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="hDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="select_image_button">
       <property name="text">
        <string>选择图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="do_button">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hBaseDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sBaseDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vBaseDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
